serving/samples/helloworld-kotlin/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: helloworld-kotlin
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: docker.io/{username}/helloworld-kotlin
```

Comment:
`docker.io/` should probably be omitted: 
I saw image pull errors with `docker.io/imikushin/helloworld-kotlin`, but `imikushin/helloworld-kotlin` worked just fine 🤷‍♂️

Reply:
It works fine for me with `docker.io`, other samples also have this.